aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthUpdateUserAttributesOptions.kt

```diff
@@ -0,0 +1,110 @@
+package com.amplifyframework.auth.cognito.options
+
+import androidx.core.util.ObjectsCompat
+import com.amplifyframework.auth.options.AuthUpdateUserAttributesOptions
+import com.amplifyframework.util.Immutable
+import java.util.Objects.requireNonNull
+
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
```

Comment:
This will need to be at the top before package

Reply:
done